packages/react-scripts/config/webpack.config.js

```diff
@@ -570,8 +547,6 @@ module.exports = function(webpackEnv) {
       // during a production build.
       // Otherwise React will be compiled in the very slow development mode.
       new webpack.DefinePlugin(env.stringified),
-      // This is necessary to emit hot updates (currently CSS only):
-      isEnvDevelopment && new webpack.HotModuleReplacementPlugin(),
```

Comment:
code reloading is using [webpack-extension-loader](https://github.com/rubenspgcavalcante/webpack-extension-reloader) not `webpack-dev-server`